src/test/kotlin/lotto/LottoTest.kt

```diff
@@ -0,0 +1,28 @@
+package lotto
+
+import lotto.domain.lotto.Lotto
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LottoTest {
+
+    @Test
+    fun `원하는 로또 번호를 생성 할 수 있다 `() {
+
+        val customLottoNumber = listOf(1, 2, 43, 42, 45, 22).sorted()
+
+        val lotto: Lotto = Lotto(customLottoNumber)
```

Comment:
```suggestion
        val lotto = Lotto(customLottoNumber)
```
타입 추론을 활용해보는 것은 어떨까요?